packages/flutter_tools/test/general.shard/analytics_test.dart

```diff
@@ -190,6 +197,56 @@ void main() {
     }, overrides: <Type, Generator>{
       Usage: () => mockUsage,
     });
+
+    testUsingContext('command sends localtime', () async {
+      const int kMillis = 1000;
+      mockTimes = <int>[kMillis];
+      // Since FLUTTER_ANALYTICS_LOG_FILE is set in the environment, analytics
+      // will be written to a file.
+      final Usage usage = Usage(versionOverride: 'test');
+      usage.suppressAnalytics = false;
+      usage.enabled = true;
+
+      usage.sendCommand('test');
+
+      final String log = fs.file('analytics.log').readAsStringSync();
+      final DateTime dateTime = DateTime.fromMillisecondsSinceEpoch(kMillis);
+      expect(log.contains(dateTime.toString()), isTrue);
+    }, overrides: <Type, Generator>{
+      FileSystem: () => memoryFileSystem,
+      SystemClock: () => mockClock,
+      Platform: () => FakePlatform(
+        environment: <String, String>{
+          'FLUTTER_ANALYTICS_LOG_FILE': 'analytics.log',
+        },
+      ),
+      Stdio: () => mockStdio,
+    });
+
+    testUsingContext('event sends localtime', () async {
```

Comment:
I'm not sure why, but https://codecov.io/gh/flutter/flutter/pull/36545/diff#D1-170 shows that adding the local time to the params isn't getting hit

Reply:
That is odd. I can't see how it wouldn't be getting covered either.